Templates/QueueTrigger-Python/function.json

```diff
@@ -7,7 +7,7 @@
       },
       {
         "type": "table",
```

Comment:
I believe @soninaren is using VS Code with the default indentation configuration (4 spaces) and that has updated our JSON files.